packages/react-components/src/components/Tooltip/UserGuide.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { ModalPortal, ModalPortalProps } from '../Modal/';
+import { ITooltipProps, Tooltip } from './Tooltip';
+import SpotlightOverlay from './SpotlightOverlay';
+import cx from 'clsx';
+import styles from './Tooltip.module.scss';
+import { ClientRectObject } from '@floating-ui/core';
+import { throttle } from '@livechat/data-utils';
+
+const spotlightPadding = 200;
+const baseClass = 'guide-tooltip';
+
+interface IOwnProps {
+  shouldSlide?: boolean;
+  className?: string;
+  disableSpotlightPointerEvents?: boolean;
+}
+
+interface IUserGuide
+  extends IOwnProps,
+    ITooltipProps,
+    Omit<ModalPortalProps, 'children'> {}
+
+export const UserGuide: React.FC<IUserGuide> = (props) => {
+  const { shouldSlide, className, parentElementName } = props;
+
+  const [parentElement, setParentElement] = React.useState<Element | null>(
+    null
+  );
+
+  const [rect, setRect] = React.useState<DOMRect | null>(null);
+
+  const handleViewportChange = () => {
+    parentElement && setRect(parentElement.getBoundingClientRect());
+  };
+
+  React.useEffect(() => {
+    if (parentElement !== null) {
+      window.addEventListener('resize', handleViewportChange);
+      window.addEventListener('scroll', handleViewportChange);
+
+      return () => {
+        window.addEventListener('resize', handleViewportChange);
+        window.addEventListener('resize', handleViewportChange);
+      };
+    }
+  }, [parentElement, parentElementName]);
+
+  React.useEffect(() => {
+    const element = document.querySelector(parentElementName);
+    setParentElement(element);
+  }, [parentElementName]);
+
+  React.useEffect(() => {
+    parentElement && setRect(parentElement.getBoundingClientRect());
+  }, [parentElement]);
+
+  return parentElement ? (
+    <ModalPortal
```

Comment:
what is the purpose of using `ModalPortal` component here? 🤔 
I gett the same working experience when changing this to just `div` element
```suggestion
    <div>
```

Reply:
Gonna be honest - I have copied it from the original design system. 
It gets even funnier because I experimented with ModalPortal in AA and switched it to div in a few places - the app works the same.